app/views/automated_tests/_test_script_result.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class='test-result'>
+  <% test_script_output = JSON.parse(test_script_result.actual_output) %>
+  <% test_script_name =
+  test_script_output['testrun']['test_script']['script_name'] %>
```

Comment:
When possible I think it looks better to access the model objects. Something like `test_script_result.test_script.script_name`?